scripts/ingests/utils.py

```diff
@@ -27,12 +28,39 @@ def disable_exception_traceback():
     sys.tracebacklimit = default_value  # revert changes
 
 
-def sort_sources(db, ingest_names, ingest_ras, ingest_decs, verbose=False, search_radius=60.):
+def sort_sources(db, ingest_names, ingest_ras, ingest_decs, search_radius=60., verbose=False):
+    '''
+
+    Parameters
+    ----------
+    db
+    ingest_names
+        Names of sources
+    ingest_ras
+        Right ascensions of sources. Decimal degrees.
+    ingest_decs
+        Declinations of sources. Decimal degrees.
+    search_radius
+        radius in arcseconds to use for source matching
+    verbose
+
+    Returns
+    -------
+    missing_sources_index
+        Indices of sources which are not in the database
+    existing_sources_index
+        Indices of sources which are already in the database
+    alt_names_table
+        Astropy table with Other Names to add to database
+
+    '''
 
     verboseprint = print if verbose else lambda *a, **k: None
 
     existing_sources_index = []
     missing_sources_index = []
+    Alt_names = namedtuple("Alt_names", "source other_name")
```

Comment:
Curious why the first entry is 'Alt_names' when it is meant to be the source name; the 'source other_name' is actually the alternate name.

Reply:
OK, clarified how this works so I'm fine with it.